tests/test_utils.py

```diff
@@ -77,3 +78,16 @@ def test_update_dict():
 
     result = update_dict(first, third)
     assert "spaniel" not in result['animals']['dogs']
+
+
+def test_check_gender():
```

Comment:
Yep, the test was maybe sowhat bad formulated. What do you think about the new version? Anyhow, realy test if both values `male` and `female` are actually returned, one should probably mock the `random.choice` method.

The thest for the `WrongArgument` is added